src/App.tsx

```diff
@@ -5,7 +5,15 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const onAddMovie = (movie: Movie) => {
+    const alreadyAdded = movies.find(m => m.imdbId === movie.imdbId);
```

Comment:
```suggestion
    const alreadyAdded = movies.find(addedMovie => addedMovie.imdbId === movie.imdbId);
```
the name of the variable should reflect its meaning